app/Http/Controllers/Auth/AuthController.php

```diff
@@ -180,13 +187,31 @@ public function getLogout()
         /*
         * Update user acive column to 0 when user successfully signout
         */
-        $logout = User::where('id', Auth::user()->id)->update(['active' => 0]);
+        $user = Auth::user();
+        
+        $data = [
+            'active' => 0
+        ];
 
+        $logout = $user->update($data);
 
         if ($logout) {
+
+            // If the user has viewed his notifications page, update his logout information.
+            if($user->has_viewed_new) {
+                $this->saveLoggedOutTime();
+            }
+
             Auth::logout();
 
             return redirect('/');
         }
     }
+
+    private function saveLoggedOutTime()
```

Comment:
**Code-Quality - Nitpick**
@andela-celisha-wigwe Every method you create should always have annotations and comments of what the function does, it goes a long way to help in the future. Please fix this here and in all other where needed,